src/services/messaging/help-messages.js

```diff
@@ -1,15 +1,17 @@
 const { CLI_NAME } = require('../config');
 
+const ENV_VAR_CMD = process.platform === 'win32' ? 'set' : 'export';
+
 exports.HELP_ENVIRONMENT_VARIABLES = `Alternatively, ${CLI_NAME} can use credentials stored in environment variables:
 
 # OPTION 1 (recommended)
-TWILIO_ACCOUNT_SID = your Account SID from twil.io/console
-TWILIO_API_KEY = an API Key created at twil.io/get-api-key
-TWILIO_API_SECRET = the secret for the API Key
+${ENV_VAR_CMD} TWILIO_ACCOUNT_SID=your Account SID from twil.io/console
+${ENV_VAR_CMD} TWILIO_API_KEY=an API Key created at twil.io/get-api-key
+${ENV_VAR_CMD} TWILIO_API_SECRET=the secret for the API Key
```

Comment:
the secret for the API Key -> the secret for the API Key, only shown when your API Key is created
or maybe
the secret for the API Key -> the secret for the API Key, created when your API Key is created

source: https://www.twilio.com/docs/iam/access-tokens#step-1-api-key